### ISSUE TITLE
Page blocks for article illustrations

### ISSUE THREAD

**Issue by sammeltassen** [digirati-co-uk/heritage.tudelft.nl#8]
Adding illustrations to articles is labour-intensive:

1. Create a new manifest in the Manifest Editor for each illustration
2. Add captions
3. Add the manifests to the `/apps/iiif/manifests/manifest-editor/illustrations` folder in this repo
4. Referencing the manifest from the article page

Page blocks already offer a friendly way to add items to the home, collection and exhibition pages. They can be added to publications too:

```mdx
<Slot name="illustrationId"></Slot>
```

In order to fully make this work for illustrations, the following enhancements are needed:

- New block template for illustrations, based on the existing illustration layout
- Ability to select a specific canvas from a manifest, and ideally a tool to crop the image if needed
- Ability to open a custom IIIF resource (in case the image is not in the collection, or from an external institution)
- Automatic ids for slots (otherwise users need to invent ids for each illustration). Ideally the deploy preview would make it possible to add new blocks between paragraphs.

**Comment by stephenwf** [comment]
Hi @sammeltassen I think this point at the end:
> Ideally the deploy preview would make it possible to add new blocks between paragraphs

We had looked at something like this together a few years ago - using the Netlify APIs. I think this would be it's own separate task, as it would also open up other editing - like Manifest Editor - and would require some extra planning.

Additionally this style of integration would be tied to the GitHub and Netlify accounts that host the static site, so it would be preferable to tackle this after #14 and avoid setting that up twice.

**Comment by sammeltassen** [comment]
@stephenwf I didn't realize this was possible and actually meant the local development preview (through the current "edit" possibility). Editing the deployment preview sounds interesting but has no priority now.

**Comment by sammeltassen** [comment]
During a discussion with a colleague, the idea came up to also allow the selection of a manifest, which would result in a slide show (like on the object pages, with the possibility to hide the thumbnails).

**Comment by stephenwf** [comment]
Ahh! That's totally fine then - yes the Page Blocks should allow for that once added to the articles. 

The Manifest page block would be totally fine too - we already have the "Box" API set up to search for Manifests, and the component for the slideshow. So matching these together would make sense so you can embed that component elsewhere.

**Comment by sammeltassen** [comment]
One more thing to add to this:

The current illustration component also supports a mixed-media canvas. Previously this would render the canvas similar to an exhibition modal, i.e. including a tour of the canvas. With the reworking of the static site, the tour functionality has disappeared, but we would like to restore this. (I'll add an example later.)
